general/development/policies/deprecation/index.md

```diff
@@ -41,13 +47,13 @@ Once it is decided that a function should be deprecated, a two-step process shou
 
```

Comment:
https://github.com/moodle/devdocs/pull/1315/files#diff-0329f1a6ba03aa03e622145c91413b7e0d55a262b5e3cd200b619496a83d0c3bR46 mentions "two-step process". I think we should update to "three-step process"